_data/jobs.yml

```
@@ -1,6 +1,6 @@
-- title: Frontend Developer HTML/CSS (m/f/d)
-  link: https://epages-gmbh.jobs.personio.de/job/224109
-  description: With your sense for modern UI design and aesthetics, you create the visual appearance of our products and drive their visual identity.
+- title: Full Stack Developer (m/f/d)
+  link: https://epages-gmbh.jobs.personio.de/job/384247
+  description: Improve our platform with your knowledge of Java or TypeScript in a cross-functional team and work on a microservice-based architecture.
```

Somehow "improve our platform" sound like it's not that good yet. I would prefer "extend" or alike and would specify the platform. I had in mind that we promoted this job before and found this description which I actually like because your suggestion would become a bit two long (5 lines) if we add "ecommerce" or alike to "platform". 

```suggestion
  description: Be part of a cross-functional team and use your Java and/or TypeScript skills to further develop our new ecommerce platform.
```

---

@cgebken I've applied your suggestion. :)